iDiff/cmd/iDiff.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"testing/runtimes-common/iDiff/differs"
+
+	"github.com/spf13/cobra"
+)
+
+// iDiff represents the iDiff command
+var iDiffCmd = &cobra.Command{
+	Use:   "iDiff [container1] [container2] [differ]",
+	Short: "Compare two images.",
+	Long:  `Compares two images using the specifed differ. `,
+	Run: func(cmd *cobra.Command, args []string) {
+		if valid, err := checkArgNum(args); !valid {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+		if args[2] == "hist" {
```

Comment:
Is this where the tool will "decide" which differ to use based on the args? It might make sense to keep a map of differ name to either instance of the differs, or entrypoint methods into them. That would make the code significantly more concise, instead of having to do a switch-esque statement here for each differ type. Just something to think about